ArduPlane/quadplane.cpp

```diff
@@ -637,6 +653,9 @@ bool QuadPlane::setup(void)
         return false;
     }
 
+    q_fwd_throttle = 0.0f;
+    q_fwd_nav_pitch_lim_cd = -aparm.angle_max;
```

Comment:
maybe a "is_active" method rather than keeping this copy.

Reply:
I don't understand.